OpenRA.Mods.RA/AI/HackyAI.cs

```diff
@@ -310,7 +310,7 @@ public bool HasMinimumProc()
 		// For mods like RA (number of building must match the number of aircraft)
 		bool HasAdequateAirUnits(ActorInfo actorInfo)
 		{
-			if (!actorInfo.Traits.Contains<ReloadsInfo>() && actorInfo.Traits.Contains<LimitedAmmoInfo>()
+			if (actorInfo.Traits.Contains<LimitedAmmoInfo>() 
```

Comment:
What I've been struggling with is that this is calling for ActorInfo rather than Actor.

Would ```!actorInfo.Traits.Get<LimitedAmmoInfo>().SelfReloads``` work?
It does compile, at least...